package trie.fifteenpuzzle.eit.eu;

public class CycleFoundException extends Exception {
	
}
